src/cluster_legacy.c

```diff
@@ -3297,7 +3297,9 @@ int clusterProcessPacket(clusterLink *link) {
             message_len = ntohl(hdr->data.publish.msg.message_len);
             channel = createStringObject((char *)hdr->data.publish.msg.bulk_data, channel_len);
             message = createStringObject((char *)hdr->data.publish.msg.bulk_data + channel_len, message_len);
+            clusterSlotStatsSetClusterMsgLength(ntohl(hdr->totlen));
```

Comment:
I guess I've been wondering it more, should we really capture pubsub traffic here? It's not capture in network bytes out and we weren't planning on capturing it in the `commandlog`. I am leaning towards it might be simpler to cut all of this and have this just represent client network traffic and ignore other overhead.